src/Compilers/VisualBasic/Portable/Symbols/Source/SourceModuleSymbol.vb

```diff
@@ -703,7 +703,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
                                 Sub()
                                     Try
                                         visitor(symbol)
-                                    Catch e As Exception When FatalError.ReportUnlessCanceled(e)
+                                    Catch e As Exception When FatalError.ReportUnlessCanceled(e, cancellationToken)
```

Comment:
these are not clear if they're correct.  can you not change our exception fitlers (unless you totally verify that the new pattern is absolutely teh one we want).

Reply:
actually, this is definitely a change in semantics and worries me.  don't do these without talking to @sharwell @tmat @jasonmalinowski  to make sure they're ok.

Reply:
reverted

Reply:
Yes, we should not change it at this point.